server/sql/prepend-i18n-bag-name.sql

```diff
@@ -0,0 +1,23 @@
+UPDATE
+    bags b
+        JOIN user_chains uc ON
+        b.user_chain_id = uc.id
+        JOIN users u ON
```

Comment:
This looks like it works but what if the bag was created by a loop host that was Dutch and hosts a Dutch loop, but the current bag holder is English or French?

A sub query is needed to get the user ID of a host of the bags loop and get the i18n for them.

Reply:
okay, that's also ok. Do I just get the host with the lowest ID?

Reply:
any host with a i18n that contains 2 letters, default to dutch if not.

Reply:
You got me there :) I misinterpreted the fields. Pushed a fix, but I assumed you mean default to English?

Reply:
Nope default to Dutch, damage control will be significantly easier

Reply:
done